src/explore-education-statistics-admin/src/pages/publication/components/PublicationForm.tsx

```diff
@@ -186,7 +196,7 @@ const PublicationForm = ({
 
               <FormFieldTextInput<FormValues>
                 name="contactTelNo"
-                label="Contact telephone number"
+                label="Contact telephone number (optional)"
```

Comment:
Just noticed this label is actually inconsistent with `PublicationContactForm`. Think we should make this consistent.

Would suggest `Contact telephone` for brevity.